bedrock/tests/test_smart_account_safe_owner_swap.rs

```diff
@@ -0,0 +1,120 @@
+use std::sync::Arc;
+
+use alloy::{
+    primitives::U256,
+    providers::{ext::AnvilApi, ProviderBuilder},
+    signers::local::PrivateKeySigner,
+    sol,
+};
+use bedrock::{
+    primitives::http_client::set_http_client,
+    smart_account::{SafeSmartAccount, ENTRYPOINT_4337},
+};
+
+mod common;
+use common::{deploy_safe, setup_anvil, AnvilBackedHttpClient, IEntryPoint};
+
+sol! {
+    /// Safe owner management interface
```

Comment:
create abi folder in primitives with sol interface + rust helper functions, its defined twice now. I have this in my PBH branch